src/repos/ArticleRepo.ts

```diff
@@ -1,28 +1,44 @@
 import Filter from 'bad-words';
 import { ObjectId } from 'mongodb';
 import { Article, ArticleModel } from '../entities/Article';
-import { DEFAULT_LIMIT, MAX_NUM_DAYS_OF_TRENDING_ARTICLES } from '../common/constants';
+import { DEFAULT_LIMIT, MAX_NUM_DAYS_OF_TRENDING_ARTICLES, IS_FILTER_ACTIVE } from '../common/constants';
 import { PublicationModel } from '../entities/Publication';
 
+function isArticleFiltered(article: Article){
+  return IS_FILTER_ACTIVE && article.isFiltered;
+}
+
 const getArticleByID = async (id: string): Promise<Article> => {
-  return ArticleModel.findById(new ObjectId(id));
+  return ArticleModel.findById(new ObjectId(id)).then(
```

Comment:
you use get getArticleByID tho.. can we keep Id and ID consistent? choose one and stick to it

Reply:
findById is a mongoose function